### PR TITLE
systemctl: replace `status` with `list-unit-files --type=service`

### PR DESCRIPTION
**Pull request by levid0s**
# Updates

Replaced `systemctl status` with `systemctl list-unit-files --type=service`, because the former is not a user-friendly way of listing the services and not compatible with grep. Both a novice user and and experienced user is much more likely to use the latter option for day-to-day operations.

# Checklist

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/systemctl.md [hunk 1]
```diff
@@ -3,9 +3,10 @@
 > Control the systemd system and service manager.
 > More information: <https://www.freedesktop.org/software/systemd/man/systemctl.html>.
 
-- Show all running services:
+- List all services (optionally filtering by state):
 
-`systemctl status`
+`systemctl list-unit-files --type=service --all`
```
Comment (mfrw) [inline pages/linux/systemctl.md:8]: Probably this is a typo IMO.
Can you please remove one of the lines here which is not appropriate

File: pages/linux/systemctl.md [hunk 2]
```diff
@@ -3,9 +3,10 @@
 > Control the systemd system and service manager.
```
Comment (mfrw) [inline pages/linux/systemctl.md:3]: Welcome to tldr @levid0s 
Can you please sign the CLA

Comment (kbdharun) [reply pages/linux/systemctl.md:3]: @levid0s Can you check this out?

**Comment by kbdharun** [comment]
> Shall we take over this PR or just close it?

Probably close in this case, as the CLA hasn't been signed yet. Feel free to apply the changes in a new PR.

**Pull request closed without merging**